.gitlab-ci.yml

```diff
@@ -841,45 +840,51 @@ test_backend_integration:
       };
       trap handle_exit EXIT
 
-    - INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
-    - echo Running backend-tests suite $INTEGRATION_TEST_SUITE
-    - cd integration/backend-tests/
+    - INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite)
 
-    # From 2.4.x on, the script would download the requirements by default
-    - if ./run --help | grep -e --no-download; then
-        RUN_ARGS="--no-download";
-      fi
+    - >
+      if [ "$INTEGRATION_TEST_SUITE" = "$TESTS_TYPE" ] || [ "$INTEGRATION_TEST_SUITE" = "all" ]; then
+        # Post job status
+        ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
+
+        echo Running backend-tests suite $INTEGRATION_TEST_SUITE
+        cd integration/backend-tests/
+
+        # From 2.4.x on, the script would download the requirements by default
+        if ./run --help | grep -e --no-download; then
+          RUN_ARGS="--no-download";
+        fi
+
+        # for pre 2.2.x releases, ignore test suite selection and just run open tests
+        if ./run --help | grep -e --suite; then
+          ./run --suite $INTEGRATION_TEST_SUITE $RUN_ARGS;
+        else
+          PYTEST_ARGS="-k 'not Multitenant'" ./run;
+        fi
 
-    # for pre 2.2.x releases, ignore test suite selection and just run open tests
-    - if ./run --help | grep -e --suite; then
-        ./run --suite $INTEGRATION_TEST_SUITE $RUN_ARGS;
+        # Always keep this at the end of the script stage
+        echo "success" > /JOB_RESULT.txt
       else
-        PYTEST_ARGS="-k 'not Multitenant'" ./run;
+        echo "skipped" > /JOB_RESULT.txt
       fi
 
-    # Always keep this at the end of the script stage
-    - echo "success" > /JOB_RESULT.txt
-
     - ")"
 
   after_script:
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
-    - if [ "$(cat /JOB_RESULT.txt)" != "success" ]; then ${CI_PROJECT_DIR}/scripts/github_pull_request_status $(cat /JOB_RESULT.txt) "Gitlab ${CI_JOB_NAME} finished" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"; fi
-
-    - find ${CI_PROJECT_DIR}/../integration/backend-tests -mindepth 1 -maxdepth 1 -name 'acceptance.*' -exec cp "{}" . \;
-    - ls ${CI_PROJECT_DIR}/../integration/backend-tests/results_*xml | xargs -n 1 -i cp {} .
-    - ls ${CI_PROJECT_DIR}/../integration/backend-tests/report_*html | xargs -n 1 -i cp {} .
-
-    - if [ "$NIGHTLY_BUILD" = "true" ]; then
-    -   ${CI_PROJECT_DIR}/scripts/mantra_post_test_results 7 nightly-$(date +%Y-%m-%d) results_backend_integration_open.xml
-    -   ${CI_PROJECT_DIR}/scripts/mantra_post_test_results 8 nightly-$(date +%Y-%m-%d) results_backend_integration_enterprise.xml
-    - fi
```

Comment:
Careful: we have removed the `mantra` integration when resolving conflicts. Please revert this part.

Reply:
When adding this, keep in mind now that there are two possible files, `results_backend_integration_open.xml` and `results_backend_integration_enterprise.xml`, so some logic is needed around it.

Reply:
@lluiscampos honestly I don't remember that I deleted it, can't get where it came from...

Reply:
Yes. This was deleted accidentally. Reverting it back.